### ISSUE TITLE
(3.0.93) it seems that a full window method pane is not updated when changed in a separate method browser

### ISSUE THREAD

**Issue by dalehenrich** [ericwinger/Jade#750]
I've just a series of episodes where I was editting a method in a separate method browser and/or debugger and the source in the full window method pane was not updated to match:
![image](https://user-images.githubusercontent.com/623951/77128027-e1945280-6a0b-11ea-9149-36962d6d914f.png)

**Comment by ericwinger** [comment]
I'm not seeing this in the latest development work, 3.1.0. Here are two screen shots. In each case, I just changed one source pane and all three updated. 

It's possible this was fixed inadvertently in 3.1.0. It's also possible that a certain type of method list browser might be the source of the problem.  

Closing for now and asking Lisa to review when 3.1.0 comes out shortly. 

![image](https://user-images.githubusercontent.com/36212516/77179768-2361df00-6a86-11ea-8bcd-782f7233db1f.png)

![image](https://user-images.githubusercontent.com/36212516/77179802-34aaeb80-6a86-11ea-960b-6d0dc9cc6ba2.png)
